spec/controllers/branches_controller_spec.rb

```diff
@@ -171,9 +171,16 @@
         expect(response).to be_success
 
         doc = Nokogiri::XML(response.body)
-        elements = doc.xpath("/Projects/Project[@name='#{repository.to_param}']")
+        elements = doc.xpath("/Projects/Project")
 
         expect(elements).to have(2).items
+
+        names = elements.map{ |e| e.attribute("name").to_s }
+        names.sort!
+
+        expected_names = ["#{repository.to_param}", "#{repository.to_param}/convergence"]
+
+        expect(names).to eq(expected_names)
```

Comment:
you can clean this up a bit with `match_array` which does not require the ordering of the arrays to match.

```
names = elements.map { |e| e.attribute("name").to_s }
expected_names = ["#{repository.to_param}", "#{repository.to_param}/convergence"]
expect(names).to match_array(expected_names)
```